libs/coin-modules/coin-canton/src/api/lastBlock.integ.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createApi } from ".";
+
+let api;
+
+describe("localnet", () => {
```

Comment:
Do you think we still need the localnet?

Reply:
it's direct implementation without gateway